packages/wonder-blocks-button/src/__docs__/accessibility.stories.mdx

```diff
@@ -0,0 +1,92 @@
+import {Meta, Story, Canvas} from "@storybook/addon-docs";
+import {StyleSheet} from "aphrodite";
+
+import Button from "@khanacademy/wonder-blocks-button";
+import {View} from "@khanacademy/wonder-blocks-core";
+
+<Meta
+    title="Navigation/Button/Accessibility"
+    component={Button}
+    parameters={{
+        previewTabs: {
+            canvas: {hidden: true},
+        },
+        viewMode: "docs",
+        chromatic: {
+            // Disables chromatic testing for these stories.
+            disableSnapshot: true,
+        },
+    }}
+/>
+
+export const Template = (args) => <Button {...args} />;
+
+## Accessibility
+
+### Labeling
+
+`Button` has an accessible label by default. The accessible name is computed from
+the `children` prop. However, `aria-label` should be used when `spinner={true}`
+to let people using screen readers that the action taken by clicking the button
+will take some time to complete.
```

Comment:
It still bums me out we don't show any visible text with `spinner={true}`.  No action req'd.